packages/shared/src/components/squads/SquadMemberShortList.tsx

```diff
@@ -1,13 +1,13 @@
 import classNames from 'classnames';
 import React, { ReactElement } from 'react';
-import { Squad, SourceMember } from '../../graphql/sources';
+import { Squad, SourceMember, PartialSquadWithRequiredFields } from '../../graphql/sources';
 import { useLazyModal } from '../../hooks/useLazyModal';
 import { LazyModal } from '../modals/common/types';
 import { ProfilePicture } from '../ProfilePicture';
 import { SimpleTooltip } from '../tooltips/SimpleTooltip';
 
 export interface SquadMemberShortListProps {
-  squad: Squad;
+  squad: PartialSquadWithRequiredFields;
```

Comment:
The name should be a bit more explicit to the use case I think.
It's very generic and doesn't say much about where it's used for.

Reply:
@rebelchris, I have changed it to `BasicSquadWithCurrentMember` currently however, if you have a better suggestion as this was the best I could think of